app/src/main/java/rectangledbmi/com/pittsburghrealtimetracker/utils/WalledInternetStrategy.java

```diff
@@ -61,4 +62,14 @@
                 })
                 .distinctUntilChanged();
     }
+
+  @Override
+  public Single<Boolean> checkInternetConnectivity(String host, int port, int timeoutInMs, ErrorHandler errorHandler) {
+    return null;
```

Comment:
I ended up not putting anything here because I didn't use this class (the library now provides an implementation of the `WalledGardenStrategy`). Maybe this class can be deleted?

Reply:
Yes. :)